kaolin/graphics/SoftRenderer.py

```diff
@@ -37,28 +37,44 @@
 # OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 # SOFTWARE.
 
+import math
+from typing import Optional
+
 import torch
 
-import kaolin as kal
 from .DifferentiableRenderer import DifferentiableRenderer
 from .Lighting import compute_ambient_light
 from .Lighting import compute_directional_light
+from .softras.rasterizer import SoftRasterizer
+from .softras.soft_rasterize import soft_rasterize
 
 
 class SoftRenderer(DifferentiableRenderer):
     r"""A class implementing the \emph{Soft Renderer}
-    from the following ICCV 2019 paper:
-        Soft Rasterizer: A differentiable renderer for image-based 3D reasoning
-        Shichen Liu, Tianye Li, Weikai Chen, and Hao Li
-        Link: https://arxiv.org/abs/1904.01786
+        from the following ICCV 2019 paper:
+            Soft Rasterizer: A differentiable renderer for image-based 3D reasoning
+            Shichen Liu, Tianye Li, Weikai Chen, and Hao Li
+            Link: https://arxiv.org/abs/1904.01786
+
+    .. note::
+        If you use this code, please cite the original paper in addition to Kaolin.
+
+        .. code-block::
+            @article{liu2019softras,
+              title={Soft Rasterizer: A Differentiable Renderer for Image-based 3D Reasoning},
+              author={Liu, Shichen and Li, Tianye and Chen, Weikai and Li, Hao},
+              journal={The IEEE International Conference on Computer Vision (ICCV)},
+              month = {Oct},
+              year={2019}
+            }
 
     """
 
     def __init__(
             self,
             image_size: int = 256,
             anti_aliasing: bool = True,
-            bg_color: torch.Tensor = torch.zeros(3),
+            bg_color: torch.Tensor = None,
             fill_back: bool = True,
             camera_mode: str = 'projection',
             K=None, rmat=None, tvec=None,
```

Comment:
there is an inconsistent style (2 arguments per line)
Is this normal that those can be `None` even if camera_mode is 'projection' ?